crates/telio-traversal/src/connectivity_check/cross_ping_check.rs

```diff
@@ -30,6 +30,7 @@ use tokio::time::{interval_at, Instant, Interval};
 use tokio::{sync::Mutex, time::MissedTickBehavior};
 
 const CPC_TIMEOUT: Duration = Duration::from_secs(10);
+const UPGRADE_TIMEOUT: Duration = Duration::from_secs(60);
```

Comment:
IMHO, maybe it is "safer" to use some bounded LRU container? 

E.g. at least theoretically storing values for fixed amount of time, allows for malicious meshnet node to spam so many session IDs, that "session_id_candidates" would explode in size.

IMHO, we can get some fixed capacity container, and store up to, for example, most recent 512 entries?

What are your thoughts about :arrow_up: ?

Reply:
I think that it's a good point and will implement it :)